specification/network/resource-manager/readme.md

```diff
@@ -115,6 +115,12 @@ suppressions:
   - code: MissingSegmentsInNestedResourceListOperation
     from: ipAddressManager.json
     reason: False alarm.
+  - code: PatchIdentityProperty
```

Comment:
Please add the path for both the suppressions instead of suppressing for the whole file.

Reply:
Hi @ramoka178 , the same pattern is followed to suppress lint diff in this file. Can you please share the doc for how to add path for suppression